client/txn.go

```diff
@@ -282,7 +288,7 @@ func (txn *Txn) Run(b *Batch) error {
 // optional, but more efficient than relying on the implicit commit
 // performed when the transaction function returns without error.
 func (txn *Txn) CommitInBatch(b *Batch) error {
-	b.calls = append(b.calls, endTxnCall(true /* commit */))
+	b.calls = append(b.calls, endTxnCall(true /* commit */, txn.Proto.GetSystemDBTrigger()))
```

Comment:
I like to use the accessors on proto fields, it allows me to not care whether we set "nullable" or not